Makefile

```diff
@@ -36,6 +45,9 @@ mock_airflow:
 mock_astro:
 	mockery --filename=Client.go --output=astro-client/mocks --dir=astro-client --outpkg=astro_mocks --name Client
 
+mock_astro_core:
```

Comment:
could we also add this to [`mock`](https://github.com/astronomer/astro-cli/pull/873/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R32) as well, so that mocking astro core is also run when someone does `make mock` because `mock` is added so that all the mocks get refreshed via a single command.